src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -151,22 +151,22 @@ const WorkflowDashboard = _.flow(
             ])
           ]),
           makeSection('Links', [
-            div({ style: { display: 'flex', flexFlow: 'row wrap', marginTop: '0.5rem' } }, [
+            div({ style: { display: 'flex', flexFlow: 'row wrap', marginTop: '0.5rem', lineHeight: '2rem' } }, [
               h(Link, {
                 ...Utils.newTabLinkProps,
                 href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
-                style: { display: 'flex' },
+                style: { display: 'flex', marginRight: '2em', alignItems: 'center' },
```

Comment:
Hmm, the problem had to do with wrapping in this section on narrow windows. If one or two of the links bumped down to the next line, they were indented by 1rem compared to the first line. But looking at it now, I'm not sure these fixes solve that either.

Since it's not technically linked to accessibility, perhaps I'll just revert this part and let the Broad team figure out how you want to handle it.
![Screen Shot 2021-06-04 at 11 37 01 PM](https://user-images.githubusercontent.com/1521111/120878941-c6b8ea00-c58d-11eb-9c6b-401d23746761.png)